PWGHF/HFL/Tasks/taskMuonCharmBeautySeparation.cxx

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file taskMuonCharmBeautySeparation.cxx.
+/// \note This workflow requires o2-analysis-fwdtrackextension as a dependency.
+/// \brief Task to estimate HF->mu in the forward direction and use DCA observable to separate b-> mu, c-> mu.
+/// \author Shreyasi Acharya <shreyasi.acharya@cern.ch>, LPC, France
+
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "ReconstructionDataFormats/TrackFwd.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfTaskMuonCharmBeautySeparation {
+  HistogramRegistry registry{"registry", {}, OutputObjHandlingPolicy::AnalysisObject, true, true};
```

Comment:
```suggestion
  HistogramRegistry registry{"registry"};
```
`AnalysisObject` is default and the last two parameters are for sorting and creating a dedicated registry directory. I doubt that you want these options.